docs/rules/prefer-number-properties.md

```diff
@@ -58,3 +68,46 @@ const foo = Number.isFinite(10);
 ```js
 if (Object.is(foo, Number.NaN)) {}
 ```
+
+```js
+const isPositiveZero = value => value === 0 && 1 / value === Number.POSITIVE_INFINITY;
+```
+
+```js
+const isNegativeZero = value => value === 0 && 1 / value === Number.NEGATIVE_INFINITY;
+```
+
+# Options
+
+Type: `object`
+
+### Infinity
```

Comment:
```suggestion
### infinity
```

The option name should use normal property casing.

Reply:
I think in this case  better use the name of the global object?

Maybe we'll need add `NaN` too

Reply:
That will just be annoying for users. `{Infinity: true}` will cause lint errors for users using JS configs. I don't think it's worth it just to match the exact casing of the global name.

Reply:
Renamed to `checkInfinity`